opentracing-util/src/main/java/io/opentracing/util/GlobalTracer.java

```diff
@@ -142,7 +171,7 @@ public static synchronized boolean registerIfAbsent(final Callable<Tracer> provi
      * @param tracer Tracer to use as global tracer.
      * @throws RuntimeException if there is already a current tracer registered
      * @see #registerIfAbsent(Callable)
-     * @deprecated Please use 'registerIfAbsent' instead which does not attempt a double registration.
+     * @deprecated Please use {@link #registerIfAbsent(Tracer)} or {@link #registerIfAbsent(Callable)} instead.
```

Comment:
Question: Can you @link from inside @deprecated in javadoc?

Reply:
yes, usually deprecated contains a link to new API.